app-text/sigil/sigil-0.9.13.ebuild

```diff
@@ -0,0 +1,93 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7} )
+
+inherit cmake-utils python-single-r1 readme.gentoo-r1 xdg-utils
+
+MY_PN="Sigil"
+
+DESCRIPTION="Sigil is a multi-platform WYSIWYG ebook editor for ePub format"
+HOMEPAGE="https://sigil-ebook.com/"
+SRC_URI="https://github.com/Sigil-Ebook/${MY_PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
Please inline the MY_PN in first part of GitHub link, so that people can easily copy-paste it into browser.